spectree/spec.py

```diff
@@ -176,6 +177,7 @@ def validate(  # noqa: PLR0913  [too-many-arguments]
         :param skip_validation: If set to `True`, the endpoint will skip
             request / response validations.
         :param operation_id: a string override for operationId for the given endpoint
+        :serialize: Always serialize the result of the endpoint.
```

Comment:
Shall we align the name? I think it's better to be `force_resp_serialize`.

BTW, we should document that it requires the `skip_validation=False`.